src/util/Twitch.cpp

```diff
@@ -0,0 +1,8 @@
+namespace chatterino {
```

Comment:
even if this works, it makes me shiver. please include the util/Twitch.hpp file here